src/test/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileReaderImplTest {
+    private static FileReader fileReader;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void initializeFields() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void read_WhitespaceInsidePath_throwException() {
+        String inputFilePath = "src/test/resources/ activities.csv";
```

Comment:
we can make constant of int, or initialize it in @Before

Reply:
In this method I want to show what happens when i put 
one of bad paths("src/test/resources/ activities.csv").